### DISCUSSION TITLE
Suggestion: Add live counter to realtime page

### DISCUSSION THREAD

**Discussion by Fawesum** [umami-software/umami#653] [General]
As per title. Feels a bit weird that I have to go out of the realtime tab to see the actual real time number in the dashboard :)

**Comment by mikecao** [comment]
The realtime numbers are based on the last 30 minutes of traffic. The current users on the dashboard are from the last 5 minutes.

**Comment by Fawesum** [comment]
Google Analytics, Matomo and others show the last 5 mins on their realtime/live page. Just think it would be neat to have both 30 mins and 5 mins on the same.
